.changes/v3.6.0/781-bug-fixes.md

```diff
@@ -0,0 +1,2 @@
+* Fixes Issue where VDC creation with storage profile `enabled=false` wasn't working. [GH-781]
```

Comment:
You have "Issue" but the issue number is missing... :)
BTW, you can remove the dot before "[GH-781]".

Reply:
updated